gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -1009,6 +1022,11 @@ gnc_plugin_page_register_ui_update (gpointer various, GncPluginPageRegister *pag
                                          "ExecAssociatedTransactionAction");
     gtk_action_set_sensitive (GTK_ACTION(action), (uri && *uri));
 
+    /* Set 'ExecAssociatedInvoice' */
+    inv = gncInvoiceGetInvoiceFromTxn(trans);
+    action = gnc_plugin_page_get_action (GNC_PLUGIN_PAGE(page), "ExecAssociatedInvoiceAction");
+    gtk_action_set_sensitive (GTK_ACTION(action), (!(!inv)));
```

Comment:
You have two ! here, I am not sure about this, not come across this in existing files.

Reply:
I remember this trick from somewhere... converts a valid invoice to TRUE and null-invoice to NULL